main.py

```diff
@@ -0,0 +1,2 @@
+if __name__ == '__main__':
+    print("hello")
```

Comment:
this is terrible

Reply:
Ivanka. Check your knowledge of python. I think this is good and you are terrible